apache2/apache2_util.c

```diff
@@ -335,12 +352,12 @@ static void send_waf_log(struct waf_lock* lock, apr_file_t** fd, const char* str
     get_field_value("[id ", "]", str1, waf_id);
     get_field_value("[line ", "]", str1, waf_line);
     get_field_value("[msg ", "]", str1, waf_message);
+    get_pcre_error_message(str1, waf_message);
     get_field_value("[data ", "]", str1, waf_data);
-	get_field_value("[ver ", "]", str1, waf_ruleset_info);
 	get_ip_port(ip_port, waf_ip, waf_port);
     get_detail_message(str1, waf_detail_message); 
     get_short_filename(waf_filename);
-    get_ruleset_type_version(waf_ruleset_info, waf_ruleset_type, waf_ruleset_version); 
+    get_ruleset_type_version(msc_waf_signature, waf_ruleset_type, waf_ruleset_version); 
```

Comment:
now that the format of the contents of this field changed, don't we also need to change something else to this new format? Or write some code to convert from the old format to the new one? I think now it's for example "OWASP_CRS/3.1.0", but this is slightly different from the old format, right?

Reply:
this format is exactly the old format, i think somehow the [ver ] get data from the same cmd